src/renderer/components/ft-list-playlist/ft-list-playlist.scss

```diff
@@ -1 +1 @@
-@use '../../scss-partials/_ft-list-item';
+@use '../../scss-partials/ft-list-item';
```

Comment:
Does this not cause issues? (The file name is still _ft-list-item)

This comment applies for all ft-list-*

Reply:
[Here's what I found on that](https://stackoverflow.com/a/27984975)